test/e2e/apimachinery/webhook.go

```diff
@@ -1150,24 +1150,15 @@ func registerValidatingWebhookForCRD(f *framework.Framework, context *certContex
 					Operations: []v1beta1.OperationType{v1beta1.Create},
 					Rule: v1beta1.Rule{
 						APIGroups:   []string{"apiextensions.k8s.io"},
-						APIVersions: []string{"*"},
```

Comment:
why this change?

Reply:
I changed this line because the "/crd" endpoint in the webhook image expects version  apiextensions.k8s.io/v1beta1. I wanted to make sure that the test wouldn't cause flaky behavior when we add v1 crd and add tests which create v1 crd objects

Reply:
wouldn't we want to be sure we updated this test and image as part of that?

Reply:
Okay I'll change this back but add a TODO about what to change to add support for v1